common/djangoapps/config_models/management/commands/populate_model.py

```diff
@@ -0,0 +1,72 @@
+"""
+Populates a ConfigurationModel by deserializing JSON data contained in a file.
+"""
+import os
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.utils import deserialize_json
+
+
+class Command(BaseCommand):
+    """
+    This command will deserialize the JSON data in the supplied file to populate
+    a ConfigurationModel. Note that this will add new entries to the model, but it
+    will not delete any entries (ConfigurationModel entries are read-only).
+    """
+    help = """
+    Populates a ConfigurationModel by deserializing the supplied JSON.
+
+    JSON should be in a file, with the following format:
+
+    { "model": "config_models.exampleconfigurationmodel",
```

Comment:
This was the format used by the Django serializer. I can try title case to see if it also works.

I also wondered why it isn't config_models_exampleconfigurationmodel, but again, I was just starting with something that I knew worked.